app/models/set_member_subject.rb

```diff
@@ -4,7 +4,6 @@ class SetMemberSubject < ActiveRecord::Base
   
   belongs_to :subject_set, counter_cache: true, touch: true
   belongs_to :subject
-  has_many :classifications
```

Comment:
We'll need these counts for simple retirement in Stargazing. I was thinking background workers that would allow over classification based on incoming rates.

When you say multi-subject classifications, what do you mean?

We could use kafka service consumer to calculate counts. Where do we store them?

My statement before was incorrect re sum (on two levels), i was assuming a similar counter feature that doesn't without a belongs to.